nat/src/stateless/mod.rs

```diff
@@ -212,6 +212,7 @@ impl StatelessNat {
                     packet.get_meta_mut().dst_vni = Some(vni);
                 }
                 if modified {
+                    packet.update_checksums();
```

Comment:
This is not correct, we should recompute the checksum for the inner packet on encap.  Can we please create an issue to do this, or fix it before merge?

Reply:
I don't think this is incorrect.  We have to decide what components are responsible for what. I agree that doing that on encap would be more generic and thought about it. However, if we go that path, we'll recompute the checksum for every encaped packet, which will unnecessarily add processing overhead. The NAT stage knows if addresses were changed or not; that's the reason for adding this there. We can, if folks so prefer, have the best of the two worlds by doing that on encap conditioned on a packet annotation that indicates if it needs to be done.